studies/demoBot/src/main/java/org/team100/frc2024/shooter/ShooterWithVerticalPivot.java

```diff
@@ -0,0 +1,72 @@
+package org.team100.frc2024.shooter;
+
+import org.team100.frc2024.shooter.drumShooter.DrumShooter;
+import org.team100.lib.dashboard.Glassy;
+import org.team100.lib.motion.components.LinearVelocityServo;
+import org.team100.lib.motion.components.OutboardGravityServo;
+import org.team100.lib.telemetry.SupplierLogger;
+import org.team100.lib.telemetry.Telemetry.Level;
+
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterWithVerticalPivot extends SubsystemBase implements Glassy{
+    
+    private final DrumShooter m_shooter;
+    private final SupplierLogger m_logger;
+    private final OutboardGravityServo m_pivot;
+    private final Servo m_indexer;
```

Comment:
now you're not using the indexer subsystem at all, you're using a WPILib Servo?  i'm confused about how this is supposed to work.  you're not supposed to use both the setAngle() and setSpeed() methods of the same servo; they apply to different kinds of hardware.

Reply:
I probably will need to test this once I get the actual build, I am not sure how it would work. The indexer subsystem is not being used but I created it in case we need to pivot (no pun intended) from a servo into a neo550 or other motor.